### PR TITLE
File separation 

### PR DESCRIPTION
**Pull request by Jerry-py**
## Description

File separation 

## Checklist

- [ ] - All code changes have been tested, if any
- [ ] - The documentation has been updated accordingly for any code changes
- [ ] - This PR fixes an already existing issue
- [x] - This PR adds a new feature (new functions, methods, classes, parameters, etc.)
- [ ] - This PR fixes a bug
- [ ] - This PR is a breaking change (removes/renames functions, parameters etc.)
- [ ] - This PR is NOT a code change (changes Documentation files, README.md, CONTRIBUTING.md etc.)

### CODE REVIEW

**Comment by Jerry-py** [comment]
Some are blank for some reason idk why

**Comment by Jerry-py** [comment]
Finished adding the code to the blank files

**Comment by Jerry-py** [comment]
Sorry for the all commits doing this in the browser

**Comment by Jerry-py** [comment]
Idk if im using circular imports or not

**Review by TheUntraceable — APPROVED** [review]
Merge Team.py with application.py, they're meant for each other.

**Comment by Jerry-py** [comment]
Ok

**Comment by TheUntraceable** [comment]
Didn't mean to approve but to request changes.

**Comment by Jerry-py** [comment]
It's fine

**Comment by Jerry-py** [comment]
@TheUntraceable Done

**Review by TheUntraceable — CHANGES_REQUESTED** [review]
I don't mean to be really really mean and want it really really one way, but Integrations are part of guilds.py, could you move it there please?

**Comment by Jerry-py** [comment]
Alr

**Review by Jerry-py — COMMENTED** [review]
Done

**Review by nav-github01001 — CHANGES_REQUESTED** [review]
@Jerry-py Can you look at my fork of this pr and try to apply changes in it?
https://github.com/nav-github01001/EpikCord.py/tree/patch-8

File: EpikCord/file.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import os
+import io 
+from typing import Optional, Union
+
+
+class File:
+    """
+    Represents a file. Sourced from Discord.py
```
Comment (TheUntraceable) [inline EpikCord/file.py:8]: Add license here please

File: EpikCord/guild.py [hunk 2]
```diff
@@ -0,0 +1,317 @@
+import datetime
+from .application import Application
+from .channel import GuildChannel, WelcomeScreen, GuildStageChannel, _figure_out_channel_type, Overwrite
```
Comment (TheUntraceable) [inline EpikCord/guild.py:3]: We don't need to import `_figure_out_channel_type` now we have `Client.util`

File: EpikCord/file.py (hunk 1 above)
Comment (Jerrydotpy) [reply EpikCord/file.py:8]: Can you do that for me?

File: EpikCord/guild.py (hunk 2 above)
Comment (Jerrydotpy) [reply EpikCord/guild.py:3]: What am i supposed to do? Its a thing a function needs but how do we remove it?

Comment (TheUntraceable) [reply EpikCord/guild.py:3]: Replace it with `Client`.`util`.`channel_from_type` or whatever I called it.

File: EpikCord/file.py (hunk 1 above)
Comment (TheUntraceable) [reply EpikCord/file.py:8]: Yea one sec

File: EpikCord/guild.py (hunk 2 above)
Comment (Jerry-py) [reply EpikCord/guild.py:3]: image

Comment (Jerry-py) [reply EpikCord/guild.py:3]: How to remove?

Comment (Jerry-py) [reply EpikCord/guild.py:3]: Like:
```py
self.client.utils._figure_out_channel_type
```
?

**Comment by Jerry-py** [comment]
@TheUntraceable Done

File: EpikCord/channel.py [hunk 3]
```diff
@@ -210,6 +210,36 @@ def __init__(self, client, data: dict):
         self.channel_id: str = data.get("channel_id")
         self.privacy_level: int = data.get("privacy_level")
         self.discoverable_disabled: bool = data.get("discoverable_disabled")
+class TextBasedChannel:
```
Comment (TheUntraceable) [inline EpikCord/channel.py:213]: This is meant to be `client.utils.channel_from_type(dict)`

**Comment by Jerry-py** [comment]
We have conflicts idk how to resolve

**Comment by Jerry-py** [comment]
So i wanna say we have to close this

**Pull request closed without merging**